samples/hello-world/src/main/java/com/samples/helloworld/app/RootActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.samples.helloworld.app
+
+import android.os.Bundle
+import android.view.ViewGroup
+import com.badoo.ribs.android.RibActivity
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.modality.BuildContext.Companion.root
+import com.samples.helloworld.R
+import com.samples.helloworld.hello_world.HelloWorld
+import com.samples.helloworld.hello_world.builder.HelloWorldBuilder
+
+class RootActivity : RibActivity() {
+
+    private val deps = object : HelloWorld.Dependency {
+        override val name: String = "world"
+    }
```

Comment:
Consider creating on the fly instead of storing a reference to it.
Alternatively, consider moving it closer to usage site (above createRib?)

Reply:
Applied in 62b2ab220124dc01d6089f1834149085ee7d8a46